web/cobrands/fixmystreet/assets.js

```diff
@@ -197,14 +197,19 @@ OpenLayers.Layer.VectorNearest = OpenLayers.Class(OpenLayers.Layer.VectorAsset,
 
     updateUSRNField: function() {
         if (this.fixmystreet.usrn) {
-            var usrn_field = this.fixmystreet.usrn.field;
-            var selected_usrn;
-            if ( this.selected_feature ) {
-                selected_usrn = this.fixmystreet.getUSRN ?
+            if (!this.fixmystreet.usrn.length) {
+                this.fixmystreet.usrn = [this.fixmystreet.usrn];
+            }
+            for (var i = 0; i < this.fixmystreet.usrn.length; i++) {
```

Comment:
This is fine, just noting as we have jQuery here anyway, it has an `$.each(array, function(idx, val) { })` function. Really should just polyfill `forEach` properly, shouldn't we, except we'd still have the jQuery fn :)